diff --git a/src/Makefile b/src/Makefile
index 69139dc..ed13308 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -209,7 +209,7 @@ install :
 # Version number calculations
 #
 ifneq ($(wildcard ../.git),)
-VERSIONS := $(shell git describe --tags --always --long --abbrev=1 --match "v*")
+VERSIONS := v1.21.1-100-$(shell git describe --tags --always --long --abbrev=1 --match "v*")
 VERSION_TUPLE	:= $(subst ., ,$(subst -, ,$(patsubst v%,%,$(VERSIONS))))
 VERSION_MAJOR	:= $(word 1,$(VERSION_TUPLE))
 VERSION_MINOR	:= $(word 2,$(VERSION_TUPLE))
